kratos/geometries/quadrilateral_3d_4.h

```diff
@@ -1539,12 +1539,8 @@ template<class TPointType> class Quadrilateral3D4
             rResult[0] += DeltaXi[0];
             rResult[1] += DeltaXi[1];
 
-            if ( norm_2( DeltaXi ) > 300 )
-            {
-                if (IsInside == false && k > 0)
-                {
-                    std::cout << "detJ =\t" << det_j << " DeltaX =\t" << DeltaXi << " stopping calculation. Iteration:\t" << k << std::endl;
-                }
+            if ( norm_2( DeltaXi ) > 300 ) {
+                KRATOS_WARNING_IF("Quadrilateral3D4", IsInside == false && k > 0) << "detJ =\t" << det_j << " DeltaX =\t" << DeltaXi << " stopping calculation. Iteration:\t" << k << std::endl;

```

Comment:
error

Reply:
or maybe we could pass some way to check if successful...

Reply:
I already said about the error. Besides, see the previous code -> std::cout

Reply:
I like the idea of @RiccardoRossi for returning if is successful 👍

Reply:
Do you mean to print "successful" or to return a boolean?, the first option will emply a lot of verbosity. This is a method quite used

Reply:
To return a bool but this would imply changes in the interface